### PR TITLE
added option for displaying the power consumption

### PR DESCRIPTION
**Pull request by hoamer**
hi,

I added an option to display also the actual power-consumption.
It can be enabled via "battery.showpowerconsumption=true".

It is just a quick implemented feature, so please review it if you have time ;)

best regards
Sebastian

### CODE REVIEW

File: bumblebee/modules/battery.py [hunk 1]
```diff
@@ -79,17 +79,24 @@ def capacity(self, widget):
...
+
         widget.set("theme.minwidth", "100%")
-        
         if bumblebee.util.asbool(self.parameter("showremaining", True))\
                 and self.getCharge(widget) == "Discharging":
             output = "{} {}".format(output, self.remaining())
 
         if bumblebee.util.asbool(self.parameter("showdevice", False)):
-            output = "{} ({})".format(output, os.path.basename(widget.name))
+            output = "{} ({})".format(output, r.read())
```
Comment (tobi-wan-kenobi) [inline bumblebee/modules/battery.py]: That is dangerous, isn't it? "r" might not be initialized (because it only is when "showpowerconsumption" is set.

**Comment by tobi-wan-kenobi** [comment]
Thanks, that's a nice addition. Looking at the code, I spotted one problem that seems to break functionality if "showpowerconsumption" is set to false, can you please verify and fix that?

Once that is done, I'll go ahead and merge, many thanks in advance!

**Comment by hoamer** [comment]
Hi,

sorry for the late response.
@tobi-wan-kenobi yes, the line was unintentionally changed. I fixed it.

I am sorry, but I can not reproduce a problem when setting "battery.showpowerconsumption" to false.
How does it affect it negatively on your site?

**Comment by tobi-wan-kenobi** [comment]
Looks good, many thanks for the contribution!

**Pull request merged**